### DISCUSSION TITLE
Wikijs postgresql compatibility

### DISCUSSION THREAD

**Discussion by karmingc** [requarks/wiki#6074] [Help / Questions] [answered]
Hi, 

I am currently using postgresql v11.x. In the documentations, it is said to be compatible with `v9.5 or later` https://docs.requarks.io/install/requirements, is it safe to assume that it is compatible with 12, 13, 14? Or has there been any specific issue with any postgresql version with a specific wiki version?

Thank you!

**Comment by NGPixel** [comment, marked as answer]
Yes it's compatible with newer versions.
